src/Const.ts

```diff
@@ -47,11 +47,7 @@ export class Const<L, A> {
 /**
  * @since 1.0.0
  */
-export const getSetoid = <L, A>(S: Setoid<L>): Setoid<Const<L, A>> => {
-  return {
-    equals: (x, y) => S.equals(x.value, y.value)
-  }
-}
+export const getSetoid = <L, A>(S: Setoid<L>): Setoid<Const<L, A>> => fromEquals((x, y) => S.equals(x.value, y.value))
```

Comment:
@sledorze I'm sorry for the inconvenience but could you please keep the `{ return ... }` form for all functions? The docs generator must improved (it's on my radar) but the following short form is not handled at the moment 

Bad

```ts
export const getSetoid = <L, A>(S: Setoid<L>): Setoid<Const<L, A>> => fromEquals((x, y) => S.equals(x.value, y.value))
```

Good

```ts
export const getSetoid = <L, A>(S: Setoid<L>): Setoid<Const<L, A>> => {
  return fromEquals((x, y) => S.equals(x.value, y.value))
}
```